libnavigation-base/api/current.txt

```diff
@@ -1156,13 +1158,6 @@ package com.mapbox.navigation.base.trip.model.eh {
   @StringDef({com.mapbox.navigation.base.trip.model.eh.RoadClass.MOTORWAY, com.mapbox.navigation.base.trip.model.eh.RoadClass.TRUNK, com.mapbox.navigation.base.trip.model.eh.RoadClass.PRIMARY, com.mapbox.navigation.base.trip.model.eh.RoadClass.SECONDARY, com.mapbox.navigation.base.trip.model.eh.RoadClass.TERTIARY, com.mapbox.navigation.base.trip.model.eh.RoadClass.UNCLASSIFIED, com.mapbox.navigation.base.trip.model.eh.RoadClass.RESIDENTIAL, com.mapbox.navigation.base.trip.model.eh.RoadClass.SERVICE_OTHER}) @kotlin.annotation.Retention(kotlin.annotation.AnnotationRetention.BINARY) public static @interface RoadClass.Type {
   }
 
-  public final class RoadName {
-    method public String getName();
-    method public boolean getShielded();
-    property public final String name;
-    property public final boolean shielded;
-  }
-
```

Comment:
Are we fine with these breaking changes?

Reply:
it's EHorizon only, and it's in beta, so I think we are ok
cc @LukasPaczos @Guardiola31337

Reply:
It's not experimental. I don't think it's OK.

Reply:
@dzinad we don't have `@Experimental` annotations for EHorizon classes, but all of them have the next comment
```
* **NOTE**: The Mapbox Electronic Horizon feature of the Mapbox Navigation SDK is in public beta
 * and is subject to changes, including its pricing.
```

Reply:
Alright then. We should have added the annotation.

Reply:
let's fix it separately https://mapbox.atlassian.net/browse/NAVSDK-835